### ISSUE TITLE
Really long in-page table of contents overflows page

### ISSUE THREAD

**Issue by codemonkey800** [chanzuckerberg/napari-hub#442]
#### Description

The in-page table of contents is too long for the page of `napari-segment-blobs-and-things-with-membranes`, and you only see the rest when you get to the bottom.

<img width="1103" alt="image" src="https://user-images.githubusercontent.com/2176050/157926424-1aa331d5-2932-42fa-bfe3-2409579eaad2.png">

#### Steps/Code to Reproduce

1. Open https://www.napari-hub.org/plugins/napari-segment-blobs-and-things-with-membranes
2. Scroll down and observe right nav

#### Expected Results

Should be able to scroll through table of contents

#### Actual Results

Overflows screen and can't scroll

**Comment by codemonkey800** [comment]
@liaprins-czi let me know if you want to solve this a different way, but I think making the in-page table of contents scrollable should be fine here

**Comment by liaprins-czi** [comment]
@codemonkey800 Yes, I think making the in-page ToC itself scrollable if it's too long is probably the best solution here!
